### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#155] [Esercizi - Bertini] [answered]
![Es 6](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/13ea8f68-5420-4ca2-ab7d-94b84ee76643)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione confermata in classe 
**(1)** I 3 giocatori hanno la stessa abilità nel gioco, infatti la probabilità di vittoria nel singolo round per qualunque giocatore è $1/2$, inoltre possiamo considerare i round eventi indipendenti. Si potrebbe pensare che facendo iniziare prima $A,B$ il giocatore $C$ sia svantaggiato, infatti $A,B$ possono vincere già al 2° round e il perdente del 1° round ha una seconda chance nel caso in cui $C$ perda al 2°, mentre $C$ può vincere nel caso migliore al 3° round e se perde non ha seconde opportunità.

**(2)** Ad ogni round $n\geq 2$ il torneo termina con probabilità $1/2$ e continua con probabilità $1/2$, quindi affinchè il tornero termini al round $n$ si devono perdere tutti i round da $2$ a $n-1$ con prob $1/2^{n-2}$ e vincere l'n-esimo con prob $1/2$:

$$\mathbb{P}(fine\ al\ round\ n)=\frac{1}{2^{n-1}}$$

> Se indichiamo con $X$ il numero di round prima che il torneo finisca (escluso il primo) ci accorgiamo che $X\sim Geom(1/2)$

$$\mathbb{P}(fine\ dopo\ n\ round)= \sum^{\infty}_{k=n+1}\frac{1}{2^{k-1}}$$


**(3)** Il gioco non è equo infatti $\mathbb{P}(A)=\mathbb{P}(B)$, mentre ci aspettiamo che $C$ avrà una probabilità minore di vittoria.
Osserviamo che $C$ può vincere solo in round multipli di $3$:
$$\mathbb{P}(vittoria\ C)=\sum^{\infty}_{i=1}\frac{1}{2^{3i-1}}$$

   Facciamo partire la sommatoria da $0$ in modo da renderla una serie geometrica:
  $$-2+2\sum^{\infty}_{i=0}\frac{1}{8^i}=-2+2\cdot\frac{8}{7}=2/7$$ 

  Quindi $\mathbb{P}(vittoria\ C)=2/7$
  Il complementare è la somma di $\mathbb{P}(A)+\mathbb{P}(B)=1-2/7=5/7$ 
  Allora $\mathbb{P}(A)=\mathbb{P}(B)=5/14$

**(4)** Analizziamo il caso in cui $n\rightarrow +\infty$:

$$\mathbb{P}(fine\ torneo)=\sum^{\infty}\_{k=2}\frac{1}{2^{k-1}}=\sum^{\infty}_{i=1}\frac{1}{2^i}=-1+\sum^{\infty}\_{i=0}\frac{1}{2^i}=-1+\frac{1}{1-\frac{1}{2}}=-1+2=1$$

Quindi sommando le probabilità che il tonero finisca al round $n=2,3,...$ osserviamo che la probabilità che termini è pari a $1$, ovvero è certo.

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/c9fe0865-b1b3-40fb-b90c-5a41f7f8d3b0)
Soluzione più laboriosa ma analoga a quella @Elia-Belli
